### PR TITLE
Changed sophia ocb

### PR DESCRIPTION
**Pull request by TokyoSU**

### CODE REVIEW

File: TombEngine/Objects/TR3/Entity/SophiaLeigh.cpp [hunk 1]
```diff
@@ -582,9 +576,9 @@ namespace TEN::Entities::Creatures::TR3
 				{
 					item.Animation.TargetState = SOPHIALEIGH_STATE_STAND;
 
-					StopSoundEffect(SFX_TR3_SOFIALEIGH_SUMMON);
-					SoundEffect(SFX_TR3_SOFIALEIGH_SUMMON_FAIL, &item.Pose);
-					SoundEffect(SFX_TR3_SOFIALEIGH_TAKE_HIT, &item.Pose);
+					StopSoundEffect(SFX_TR3_SOFIALEE_SUMMON);
+					SoundEffect(SFX_TR3_SOFIALEE_SUMMON_FAIL, &item.Pose);
+					SoundEffect(SFX_TR3_SOFIALEE_TAKE_HIT, &item.Pose);
```
Comment (Adngel) [inline TombEngine/Objects/TR3/Entity/SophiaLeigh.cpp:581]: I'm getting something lost here.

In the new Sophia's branch, is renaming the sounds from "SFX_TR3_SOFIALEIGH_SUMMON" to "SFX_TR3_SOFIALEE_SUMMON" but when I check in my current master branch, I'm finding "SFX_TR3_SOFIALEIGH_SUMMON"

Why is returning to the Lee?

Comment (Kubsy) [reply TombEngine/Objects/TR3/Entity/SophiaLeigh.cpp:581]: hmm merge error?

Comment (TokyoSU) [reply TombEngine/Objects/TR3/Entity/SophiaLeigh.cpp]: Fixed thanks :)

**Review by Kubsy — APPROVED** [review]
I see nothing wrong with the code (apart from that sfx enum which was fixed as we speak) so yea I approve :)

**Pull request closed without merging**